src/components/transaction-list/transaction-list.vue

```diff
@@ -164,6 +182,26 @@ export default Vue.extend({
         });
       }
       return date.format('DD MMMM');
+    },
+    async infiniteHandler($state: StateChanger): Promise<void> {
```

Comment:
```suggestion
    async infiniteHandler(state: StateChanger): Promise<void> {
```

I don't see the need in `$`-prefixed variable name. Even if this name/prefix is semantic, it shouldn't work because of scope shadowing